internal/env/load.go

```diff
@@ -2,17 +2,37 @@ package env
 
 import (
 	"bufio"
+	"log"
 	"os"
+	"path/filepath"
 	"strings"
+
+	"github.com/quickfeed/quickfeed/kit/sh"
 )
 
-var dotEnvPath = ".env"
+const dotEnvPath = ".env"
+
+var quickfeedRoot string
+
+func init() {
+	quickfeedRoot = os.Getenv("QUICKFEED")
+	if quickfeedRoot == "" {
+		out, err := sh.Output("go list -m -f {{.Dir}}")
```

Comment:
@r0qs Pointed out that this may not work if compiled into a binary, unless the go command is actually installed. This should not be a problem for now, as long as we run it from a system with go installed. However, we might consider in the future to replace `go list` with a library function to scan for the `go.mod` file. Not sure if there is a library func for this in e.g., `x/tools` or some other place.